README.md

```diff
@@ -1,2 +1,54 @@
 # Capstone-project
-In this capstone project I'm going to build, it is based on an online website for a concert.
+
+A website designed for comedy show
+
+## Built With
+
+- HTML, CSS and JS
+
+## Live Demo
+
+ [here](/)
```

Comment:
- [ ] Please 🙏🏽 deploy your application with Github and add the corresponding path to fix the link.